core-api/src/main/java/org/silverpeas/core/persistence/datasource/OperationContext.java

```diff
@@ -53,6 +55,8 @@
 
   // The user
   private User user = null;
+  private List<PersistenceOperation> persistenceOperations = new ArrayList<>();
+  private List<SilverpeasJpaEntity> entities = new ArrayList<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "entities" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1068)